build-system/npm-publish/write-package-files.js

```diff
@@ -104,9 +104,10 @@ async function getDescription() {
 
 /**
  * Write package.json
+ * @param {boolean} useBentoCore
  * @return {Promise<void>}
  */
-async function writePackageJson() {
+async function writePackageJson(useBentoCore = false) {
```

Comment:
This function is only called in a single place, can this become a mandatory argument? Alternatively making the argument a typed object to increase readability of the boolean parameter.
```suggestion
 * @param {{
    useBentoCore: boolean,
  }} options
* @return {Promise<void>}
*/
async function writePackageJson({useBentoCore}) {
```

Reply:
updated